frontend/packages/ceph-storage-plugin/src/features.ts

```diff
@@ -48,6 +49,7 @@ const OCS_FEATURE_FLAGS = {
   [GUARDED_FEATURES.OCS_ARBITER]: 'arbiter',
   [GUARDED_FEATURES.OCS_KMS]: 'kms',
   [GUARDED_FEATURES.OCS_FLEXIBLE_SCALING]: 'flexible-scaling',
+  [GUARDED_FEATURES.OCS_POOL_MANAGEMENT]: 'pool_management',
```

Comment:
we should be consistent with naming. We are using `hyphen` for flexible-scaling, so lets stick with this and make sure the same is reflected in operator and  README.
FYI please update README of ceph package with added feature guard.
Thanks
```suggestion
  [GUARDED_FEATURES.OCS_POOL_MANAGEMENT]: 'pool-management',
```

Reply:
ack